packages/@aws-cdk/aws-apigatewayv2-integrations/lib/alb-integration.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpMethod, IVpcLink } from '@aws-cdk/aws-apigatewayv2';
+import * as elbv2 from '@aws-cdk/aws-elasticloadbalancingv2';
+import { HttpProxyPrivateIntegration } from './http-private';
+
```

Comment:
apigatewayv2 supports both HTTP API and Websocket APIs. As far as I can tell, all of this new integration is aimed at HTTP APIs only and will not work for websocket.

Given this, move all of these under `lib/http/` folder and perhaps add `Http` as a prefix so that these classes are namespaced and won't collide with potential future integrations for Websocket.